tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -481,13 +484,8 @@ suite "Onchain group manager":
     except Exception, CatchableError:
       assert false, "exception raised: " & getCurrentExceptionMsg()
 
-    await fut
+    check await fut.withTimeout(5.seconds)
```

Comment:
ah, this is too slow. Isn't there a way to speed up the validation?

edited: okay, I guess the `5.seconds` wait is the worst scenary and it will fail the test in such case